src/codegen.cpp

```diff
@@ -6958,9 +6958,6 @@ extern "C" void *jl_init_llvm(void)
     eb  .setEngineKind(EngineKind::JIT)
         .setMCJITMemoryManager(std::unique_ptr<RTDyldMemoryManager>{createRTDyldMemoryManager()})
         .setTargetOptions(options)
-#if (defined(_OS_LINUX_) && defined(_CPU_X86_64_))
-        .setRelocationModel(Reloc::PIC_)
-#endif
```

Comment:
since this defaults to `PIC_` on many platforms, shouldn't this be forced off to static here instead?